rpc/src/rpc_service.rs

```diff
@@ -270,14 +271,24 @@ impl RequestMiddleware for RpcRequestMiddleware {
                 .into()
         } else if self.is_file_get_path(request.uri().path()) {
             self.process_file_get(request.uri().path())
-        } else if request.uri().path() == "/health" {
-            hyper::Response::builder()
-                .status(hyper::StatusCode::OK)
-                .body(hyper::Body::from(self.health_check()))
-                .unwrap()
-                .into()
         } else {
-            request.into()
+            match request.uri().path() {
+                "/health" => hyper::Response::builder()
+                    .status(hyper::StatusCode::OK)
+                    .body(hyper::Body::from(self.health_check()))
+                    .unwrap()
+                    .into(),
+                "/prometheus" => hyper::Response::builder()
```

Comment:
The standard location for exposing Prometheus metrics is at `/metrics`.
```suggestion
                "/metrics" => hyper::Response::builder()
```